src/validators/TupleValidator.ts

```diff
@@ -0,0 +1,41 @@
+import type { BaseError, IConstraint } from '..';
```

Comment:
Please remove all the `'..'` type imports and target the actual source files instead of the re-exports in `index.ts`

Reply:
Vsc auto import. I'll fix it tomorrow

Reply:
Figured as much yeah